nexus/src/app/sagas/instance_migrate.rs

```diff
@@ -136,6 +141,40 @@ async fn sim_allocate_propolis_ip(
     allocate_sled_ipv6(&opctx, sagactx, "dst_sled_uuid").await
 }
 
+/// Add V2P mappings for the destination instance
+async fn sim_v2p_ensure(
+    sagactx: NexusActionContext,
+) -> Result<(), ActionError> {
+    let osagactx = sagactx.user_data();
+    let params = sagactx.saga_params::<Params>()?;
+    let opctx = crate::context::op_context_for_saga_action(
+        &sagactx,
+        &params.serialized_authn,
+    );
+    let (instance_id, _) =
+        sagactx.lookup::<(Uuid, InstanceRuntimeState)>("migrate_instance")?;
+
+    // TODO-performance the instance_put in sim_instance_migrate will *start* a
+    // migration, but the source and destination propolis servers will perform
+    // it asynchronously. Updating the mappings here will briefly "disconnect"
+    // the source instance in the sense that it will be able to send packets out
```

Comment:
Is this true? We haven't yet changed the `sled_id` for the instance which happens in the next stage (`sim_instance_migrate`). I know migration is in flux so doesn't need to be a blocker for this PR

Reply:
Yeah, you're right - the comment is still true but `create_instance_v2p_mappings` needs the updated `sled_id`. I moved the v2p ensure step to after the migration start step, and updated the comment a bit in b459ce6e.